lib/cli-entry.js

```diff
@@ -29,7 +29,7 @@ module.exports = async (process, validateEngines) => {
   process.on('unhandledRejection', exitHandler)
 
   // It is now safe to log a warning if they are using a version of node that is not going to fail on syntax errors but is still unsupported and untested and might not work reliably. This is safe to use the logger now which we want since this will show up in the error log too.
-  if (!satisfies(validateEngines.node, validateEngines.engines)) {
+  if (!satisfies(validateEngines.node, validateEngines.engines, { includePrerelease: true })) {
```

Comment:
This doesn't seem to work for the bottom of each comparator (`X.0.0`), eg:

```js
> satisfies('18.0.0', '^14.17.0 || ^16.13.0 || >=18.0.0', { includePrerelease: false })
true
> satisfies('18.0.0-pre.0', '^14.17.0 || ^16.13.0 || >=18.0.0', { includePrerelease: false })
false
```

Reply:
I think this is an issue with our engines. They should probably be `^14.17.0 || ^16.13.0 || >=18`.

```
> satisfies('18.0.0-pre.0', '^14.17.0 || ^16.13.0 || >=18', { includePrerelease: true })
true
```

Reply:
I created a template-oss PR to fix this implement this at the source:

https://github.com/npm/template-oss/pull/317